### PR TITLE
Use react router to display Nav items

### PR DESCRIPTION
**Pull request by Ceemos96**
### In this milestone, I :
- Created a website consisting of 3 pages: Home, Calculator, and Quote.
- Created the general layout to match the design wireframe.
- Made sure that navigation links for all 3 routes are displayed on each page.
- Used React router to make the routes work.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+import Nav from './components/Nav';
 
 const App = () => (
-  <Calculator />
+  <div>
+    <Nav />
+    <Routes>
+      <Route path="calculator" element={<Calculator />} />
+      <Route path="quotes" element={<Quotes />} />
+      <Route path="home" element={<Home />} />
```
Comment (TedLivist) [inline src/App.js:14]: - [ ] When I load your app, it displays nothing on the root app (see image). But the Home component should be your root path. You can fix this by changing the path "home" to "/", like so:

  ```
  <Route path="/" element={<Home />} />
  ```

  home-j

File: src/components/Nav.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import { Link } from 'react-router-dom';
+
+const Nav = () => (
+  <div className="nav-wrapper">
+    <Link className="heading-text" to="/home">
+      Math Magicians
+    </Link>
+    <ul className="nav-links">
+      <li className="nav-link">
+        <Link activeStyle={{ color: 'blue' }} to="home">
```
Comment (TedLivist) [inline src/components/Nav.js:11]: - [ ] We also need to make the navbar home menu match the route in the App component above. Kindly change this line to this:

  ```
  <Link activeStyle={{ color: 'blue' }} to="/">
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Ceemos96,

Good job so far!
✔️ Good documentation
✔️ Use of RRD for SPA implementation
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Nav.js (hunk 2 above)
Comment (Ceemos96) [reply src/components/Nav.js]: Hi @TedLivist ,

I have effected the requested changes, but it seems you approved the project already so I am unable to request another review through my dashboard.

Kindly review and comment your approval if the changes I made are satisfactory.

Cheers 🥂

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Ceemos96,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**